src/client/CourseRequestForm/index.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+// before mounting your React application:
+import { theme } from '@instructure/canvas-theme'
+theme.use();
+
+import { Button } from '@instructure/ui-buttons'
+import { Checkbox, CheckboxGroup } from '@instructure/ui-checkbox'
+import { Flex } from '@instructure/ui-flex'
+import { FormFieldGroup } from '@instructure/ui-form-field'
+import { Table } from '@instructure/ui-table';
+import { TextInput } from '@instructure/ui-text-input'
+
+const CourseRequestForm = ({courses}) => {
+    const [emailToggleValue, toggleEmail] = useState(false);
+    const handleEmailToggle = (() => {
+        toggleEmail(!emailToggleValue);
+    });
+    const CrossListings = ({term, courseInfoList}) => {
+        if (courseInfoList.length !== 0) {
+            return (
+                courseInfoList.map((courseInfo, index) => (
+                    <Checkbox
+                        key={index}
+                        size='small'
+                        label={ term + '-' + courseInfo.classID + ' (' + courseInfo.subjectAreaCode + ' ' + courseInfo.courseCatalogNumberDisplay + '-' + courseInfo.classNumber + ')' }
+                        defaultChecked
+                    >
+                    </Checkbox>
+                ))
+            );
+        }
+        return null;
+    }
+    const SettingCheckbox = ({isChecked}) => (
+        isChecked ? <Checkbox label='' size='small' defaultChecked/> : <Checkbox label='' size='small'/>
```

Comment:
@kubilayagi One of the props that the Checkbox component has is defaultChecked and it is a bool. Can't you just have:

<Checkbox label='' size='small' defaultChecked={course.emailInstructors} />

and

<Checkbox label='' size='small' defaultChecked={course. sendUrl} />

See https://instructure.design/#Checkbox

Reply:
@rlorenzo I was having trouble with this. I tried doing it the way you suggested, but even when the passed value was false, it would still be checked. I even tried `<Checkbox ... defaultChecked='false' />` but it was still checked. Maybe I am missing something.